omod/src/main/java/org/openmrs/module/reporting/web/reports/ManageReportsController.java

```diff
@@ -137,6 +140,10 @@ public String editReportDesign(ModelMap model,
             url.append("&returnUrl=" + returnUrl);
         }
 		return "redirect:"+url.toString();
+    }catch(NullPointerException e){
```

Comment:
it was only here , where an empty parameter passed to the controller throws a NPE